.github/workflows/version_bump.yml

```diff
@@ -16,7 +16,7 @@ on:
 jobs:
   build:
     name: version bump
-    runs-on: ubuntu-20.04-xl
+    runs-on: ubuntu-22.04-xl
```

Comment:
@dsherret why is this using xl?

Reply:
We could downgrade it. Same for start_release.yml